### PR TITLE
Cleanbuild

### PR DESCRIPTION
**Pull request by paulvanbrenk**
Get rid of the powershell build script and move to a msbuild solution.

### CODE REVIEW

File: Build/Common.Build.settings [hunk 1]
```diff
@@ -98,23 +98,9 @@
 
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
 
-    <DevEnvDir Condition="'$(DevEnvDir)' == ''">$([MSBuild]::GetRegistryValueFromView('HKEY_LOCAL_MACHINE\Software\Microsoft\VisualStudio\$(VSTarget)', 'InstallDir', null, RegistryView.Registry32))\</DevEnvDir>
```
Comment (paulvanbrenk) [inline Build/Common.Build.settings]: This doesn't work for VS 2017, so we're using the $(VSInstallRoot) environment variable which is set.

File: Build/Common.Build.settings [hunk 2]
```diff
@@ -98,23 +98,9 @@
 
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
 
-    <DevEnvDir Condition="'$(DevEnvDir)' == ''">$([MSBuild]::GetRegistryValueFromView('HKEY_LOCAL_MACHINE\Software\Microsoft\VisualStudio\$(VSTarget)', 'InstallDir', null, RegistryView.Registry32))\</DevEnvDir> 
-    <VSSDKDir>$([MSBuild]::GetRegistryValueFromView('HKEY_LOCAL_MACHINE\Software\Microsoft\VisualStudio\VSIP\$(VSTarget)', 'InstallDir', null, RegistryView.Registry32))\</VSSDKDir>
+    <DevEnvDir Condition="'$(DevEnvDir)' == ''">$(VsInstallRoot)\Common7\IDE\</DevEnvDir>
```
Comment (billti) [inline Build/Common.Build.settings]: Sure VsInstallRootr is always set? It's not on __my machine__

File: Nodejs/Product/TestAdapter/TestAdapter.csproj [hunk 3]
```diff
@@ -9,8 +10,12 @@
     <OldToolsVersion>4.0</OldToolsVersion>
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
+    <GeneratePkgDefFile>false</GeneratePkgDefFile>
```
Comment (billti) [inline Nodejs/Product/TestAdapter/TestAdapter.csproj]: Does this not generate the pkgdef? If not, what does? You added an attribute above for use in pkgdef generation.

File: Nodejs/Product/TestAdapterVsix/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
+﻿using System.Reflection;
```
Comment (billti) [inline Nodejs/Product/TestAdapterVsix/Properties/AssemblyInfo.cs]: What does this file do? Do we need it?

**Pull request merged**